vm/systemSmartContracts/staking.go

```diff
@@ -200,24 +200,20 @@ func (r *stakingSC) unBound(args *vmcommon.ContractCallInput) vmcommon.ReturnCod
 
 	err := json.Unmarshal(data, &registrationData)
 	if err != nil {
-		log.Error("unmarshal error in unBound function of staking smart contract " + err.Error())
-		JLS
 		log.Debug("unmarshal error on finalize unstake function",
 			"error", err.Error(),
 		)
 		return vmcommon.UserError
 	}
 
 	if registrationData.Staked == true || registrationData.UnStakedNonce <= registrationData.StartNonce {
-		log.Error("unBound is not possible for address with is staked")
-		JLS
-		log.Debug("validator did not unstaked yet")
+		log.Debug("unBound is not possible for address with is staked")
```

Comment:
please refactor this and include in the log also the second condition which could lead to an error

Reply:
rephrased